wordpress-persistent-disks/mysql.yaml

```diff
@@ -17,6 +17,8 @@ spec:
       containers:
         - image: mysql:5.6
           name: mysql
+          args:
+            - "--ignore-db-dir=lost+found"
```

Comment:
this fix is only necessary if you're gonna change image to mysql:5.7.

this manifest actually works just fine with the currently listed image. the bug only happens when people try to use a newer image.

Reply:
OK. I've changed the image to 5.7.